pkg/query-service/app/dashboards/model.go

```diff
@@ -289,6 +289,10 @@ func GetDashboard(ctx context.Context, uuid string) (*Dashboard, *model.ApiError
 		return nil, &model.ApiError{Typ: model.ErrorNotFound, Err: fmt.Errorf("no dashboard found with uuid: %s", uuid)}
 	}
 
+	if dashboard.Data["title"] == "Ingestion" && dashboard.Data["description"] != nil {
+		dashboard.Data["description"] = "This dashboard is deprecated. Please use the new dashboard which is Ingestion V2. Or add it from here https://github.com/SigNoz/dashboards/blob/main/signoz-ingestion-analysis/signoz-ingestion-analysis-v1.json. " + dashboard.Data["description"].(string)
```

Comment:
```suggestion
		dashboard.Data["description"] = "This dashboard is deprecated. Please use the new Ingestion V2 dashboard. " + dashboard.Data["description"].(string)
```

Since we are adding an updated one ourselves, I think this is redundant.